### PR TITLE
Map cleanup

### PR DESCRIPTION
**Pull request by RadiantFlash7**
Map cleanup run through.
Removes unconnected wires, dead ending pipes, and random pipes left over on the map.
Also fixes red armoury opening on non-red alert situations. /No more going to the tactical armoury on a medical alert./
Also fixes anouncement for spiders being janky.

### CODE REVIEW

File: code/modules/events/spider_migration_yw.dm [hunk 1]
```diff
@@ -11,9 +11,9 @@
 /datum/event/spider_migration/announce()
 	var/announcement = ""
 	if(severity == EVENT_LEVEL_MAJOR)
-		announcement = "Massive breach of [station_name()] walls detected. Unknown biological entities has been detected near within perimeter. Heavily armed Security Patrol required."
+		announcement = "Massive breach of [station_name()] walls detected. Unknown biological entities has been detected near within walls perimeter. Heavily armed Security Patrol required."
```
Comment (KillianKirilenko) [inline code/modules/events/spider_migration_yw.dm:14]: should probably be something like "...walls detected. Unknown biological entities have been detected within outpost perimeter."

File: code/modules/events/spider_migration_yw.dm [hunk 2]
```diff
@@ -11,9 +11,9 @@
...
 	else
-		announcement = "Unknown biological [spawned_spider.len == 1 ? "entity has" : "entities have"] bypassed [station_name()]'s perimeter walls, Security Patrol."
+		announcement = "Unknown biological [spawned_spider.len == 1 ? "entity has" : "entities have"] bypassed [station_name()]'s perimeter walls, Security Patrol reccomended."
```
Comment (KillianKirilenko) [inline code/modules/events/spider_migration_yw.dm:16]: recommended. one c, two ms.

File: code/modules/events/spider_migration_yw.dm (hunk 1 above)
Comment (RadiantFlash7) [reply code/modules/events/spider_migration_yw.dm:14]: You're right. Thats what I get for looking at it at 2 am.

File: code/modules/events/spider_migration_yw.dm (hunk 2 above)
Comment (RadiantFlash7) [reply code/modules/events/spider_migration_yw.dm]: Fixed!

**Comment by Repede** [comment]
is this ready to merge?

**Comment by izac112** [comment]
curious about that too.

**Pull request merged**